src/main/webapp/app/delete-dialog/delete-dialog-service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
```

Comment:
Don't know where to put this best, but the name of the file is wrong:
`delete-dialog.service.ts`

Reply:
True, typo here